Source: Upstream, Gentoo bug 191154 
Upstream: Yes
Reason: CVE-2007-4565, NULL pointer dereference.
	http://fetchmail.berlios.de/fetchmail-SA-2007-02.txt

Index: sink.c
===================================================================
--- fetchmail-6.3.8/sink.c	(revision 5118)
+++ fetchmail-6.3.8/sink.c	(revision 5119)
@@ -262,7 +262,7 @@
     const char *md1 = "MAILER-DAEMON", *md2 = "MAILER-DAEMON@";
 
     /* don't bounce in reply to undeliverable bounces */
-    if (!msg->return_path[0] ||
+    if (!msg || !msg->return_path[0] ||
 	strcmp(msg->return_path, "<>") == 0 ||
 	strcasecmp(msg->return_path, md1) == 0 ||
 	strncasecmp(msg->return_path, md2, strlen(md2)) == 0) 
